src/pages/ScoreReport.vue

```diff
@@ -105,7 +95,16 @@
             @filter="onFilter($event)"
             @export-all="exportAll"
             @export-selected="exportSelected"
-          />
+          >
+          <label for="ms-columns" class="view-label">View</label>
+          <PvDropdown
+            id="view-columns"
```

Comment:
Shouldn't this `id` be an `input-id` and should it match the `for` param of the `label` element?
```suggestion
          <label for="view-columns" class="view-label">View</label>
          <PvDropdown
            input-id="view-columns"
```